JSDemos/Demos/List/GroupedList/description.md

```diff
@@ -1 +1,11 @@
-The **List** UI component can display items in&nbsp;groups if&nbsp;they are arranged so&nbsp;in&nbsp;the data source. Groups have captions, which are customized in&nbsp;this demo.
\ No newline at end of file
+It is possible to display the **List** in groups under the following conditions:
+
+- The **List** data passed to the **dataSource** is structured so that each group is an object that contains two fields: **key** and **items**. The **key** field represents a group header, and the **items** field represents an array of group items.
+
+    In cases when the data source doesn't conform to the structure specified above, use the [DataSource](https://js.devexpress.com/Documentation/ApiReference/Data_Layer/DataSource/)'s [group](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#group) property to define the data field to group against. If data objects are grouped but use other field names, implement the DataSource's [map](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#map) function to create **key** + **items** field mappings. Refer to the following help topic for more information: [Item Mapping](/Documentation/Guide/Data_Binding/Data_Layer/#Reading_Data/Data_Transformation/Item_Mapping).
```

Comment:
Now switching to regular data sources where we can specify a grouping field. But note below - I only assume what the list will display if we use such a data source. You'll need to fill in the blanks. 

```suggestion
### Grouping Field

The data source can contain a plain list of objects with multiple fields. Assign a field to the [DataSource](https://js.devexpress.com/Documentation/ApiReference/Data_Layer/DataSource/)'s [group](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#group) property to create a hierarchical display. Parent items will display unique values from the specified field. Child items will display ... 
```

Reply:
I'm not quite clear on what are you describing here.
Are you talking about the case when the data looks like so:
`var data = [
    { name: "Amelia", birthYear: 1991, gender: "female" },
    { name: "Benjamin", birthYear: 1983, gender: "male" },
    { name: "Andrew", birthYear: 1991, gender: "male" },
    { name: "Daniela", birthYear: 1983, gender: "female" },
    { name: "Lee", birthYear: 1983, gender: "male" },
    { name: "Betty", birthYear: 1983, gender: "female" }
];
var dataSource = new DevExpress.data.DataSource(data);`

Basically, are you talking about the scenario where we use dataSource's group() method?
`dataSource.group("birthYear");
dataSource.load().done(function(result) {
    //'result' contains the loaded array
});`

Reply:
Yes, that's what I thought you meant by "use the [DataSource]'s [group] property to define the data field to group against."

If that group property doesn't work like that, then how does it work?